docs/data/hubble/analytics-platforms.mdx

```diff
@@ -7,14 +7,16 @@ title: Ecosystem Data Analytics Platforms
 
 The following is a list of data analytics platforms that make a complete historical record of Pubnet Stellar network data available.
 
-| Platform | Access | Dashboard |
-| --- | --- | --- |
-| [Dune](https://docs.dune.com/home) | [Access](https://dune.com/) | [Stellar Dashboard](https://dune.com/scoffie/stellar) |
-| [Ortege](https://www.ortege.ai/) | [Access](https://app.ortege.ai/login/) |  |
```

Comment:
Do we still use Ortega?

Reply:
We do not use it, but I would still list on our site because they were winners in several SCF rounds. I'd prefer we add more than subtract

Reply:
So others that we're missing here: Artemis, Nansen, Flipside, DeFiLlama